proto/warden/async/v1beta1/task.proto

```
@@ -15,6 +15,10 @@ syntax = "proto3";
 
 package warden.async.v1beta1;
 
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)